service/config_provider_test.go

```diff
@@ -55,7 +56,10 @@ func (m *mockProvider) Retrieve(_ context.Context, _ string, watcher config.Watc
 }
 
 func (m *mockProvider) Scheme() string {
-	return "mock"
+	if m.shceme == "" {
```

Comment:
```suggestion
	if m.scheme == "" {
```

Reply:
I like how IDE was smart enough so if I wrote a typo once, helped me to survive with that :)))